### ISSUE TITLE
DeepinOS sound when the OS boots up ... mismatch in timing of the sound played against the login screen visibility ...

### ISSUE THREAD

**Issue by aiamuzz** [linuxdeepin/developer-center#1399]
Hi,

I have freshly installed DeepinOS 15.10.2 and have been testing stuff one by one ...

Its nice that you've implemented the sound that is supposed to play out when the login windows is displayed ... however this imlementation seems off ... as the sound plays 10-15 seconds before the actual login screen is displayed on the monitor.

Is there a way you can sync the visual cue with the audio cue ... i'e ... play the sound only after the Login Screen actually is displayed on the screen ?

Cheers.

**Comment by justforlxz** [comment]
Sorry, this issue will be closed soon. If it is necessary to discuss it again, please create a new issue.
